src/onegov/core/utils.py

```diff
@@ -416,7 +417,22 @@ def linkify(text: str, escape: bool = True) -> str:
     if not text:
         return text
 
-    linkified = linkify_phone(bleach.linkify(text, parse_email=True))
+    long_top_level_domains = ['.agency']
+
+    # bleach.linkify supports only a fairly limited amount of tlds
+    if any(domain in text for domain in long_top_level_domains):
+        if '@' in text:
+            linkified = str(
+                Markup('<a href="mailto:{text}">{' 'text}</a>').format(
```

Comment:
Looks like you accidentally inserted a `' '` here, it won't change the outcome, since string literals will implicitly concat with each other, but it definitely doesn't seem intentional.